src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -161,14 +171,46 @@ public static String samsEqualStringent(final File actualSam, final File expecte
         return compareReads(actualSam, expectedSam, validation, reference);
     }
 
+    public static String samsEqualStringent(final Path actualSam, final Path expectedSam, final ValidationStringency validation, final Path reference) throws IOException {
```

Comment:
Can you make the file overload of this method call into the Path version?

Reply:
Also add javadoc

Reply:
Certainly! Done.